src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -343,6 +362,68 @@ public BiomeGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 
 		pane5.add("Center", PanelUtils.totalCenterInPanel(spawnproperties));
 
+
+
+		JPanel multiproperties = new JPanel(new GridLayout(8, 3, 0, 2));
+		multiproperties.setOpaque(false);
+
+
+		//HERE BEGINS MULTI NOISE, PREPARE FOR FREEDOM!!!

```

Comment:
Remove this comment and other similar comments (I'm saving you from klemen)

Reply:
This was fixed in "First Fix"